### PR TITLE
Feature add

### PR DESCRIPTION
**Pull request by mrmuli**

### CODE REVIEW

File: App/models.py [hunk 1]
```diff
@@ -4,39 +4,36 @@
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy import Table, Column, Integer, String, Boolean
 
-
-engine = create_engine('sqlite:///amity.db', echo = False)
+# engine = create_engine('sqlite:///amity.db', echo = False)
```
Comment (njirap) [inline App/models.py]: commented code is bad for your health..

File: App/person.py [hunk 2]
```diff
@@ -124,45 +137,58 @@ def reallocate_person(self, person_id, room_name):
...
             # remove person in the already allocated room
-            for room in Room.total_rooms.keys():
-                for occupant in Room.total_rooms[room]:
-                    if person_id == occupant:
-                        Room.total_rooms[room].remove(person_id)
+            # for room in Room.total_rooms.keys():
+            #     if room != room_name:
+            #         for occupant in Room.total_rooms[room]:
+            #             if person_id == occupant:
+            #                 Room.total_rooms[room].remove(person_id)
```
Comment (njirap) [inline App/person.py:194]: don't do commented code..

File: App/models.py (hunk 1 above)
Comment (mrmuli) [reply App/models.py]: noted

File: App/person.py (hunk 2 above)
Comment (mrmuli) [reply App/person.py:194]: noted

**Pull request merged**